scripts/decrypt.sh

```diff
@@ -9,6 +9,7 @@ set -o nounset
 set -o xtrace
 
 if [[ "${TESTING_PROFILE}" == *"integration-tests"* ]] || [[ "${TESTING_PROFILE}" == "automated-review" ]]; then
-    openssl aes-256-cbc -K $encrypted_5277f413f627_key -iv $encrypted_5277f413f627_iv -in secrets.tar.enc -out secrets.tar -d
+    openssl aes-256-cbc -K $encrypted_4e4ba0f4c3c2_key -iv $encrypted_4e4ba0f4c3c2_iv -in secrets.tar.enc -out secrets.tar -d
     tar xvf secrets.tar
+    mv dockstore-integration-testing/src/test/resources/dstesting_pcks8.pem /home/travis/dstesting_pcks8.pem
```

Comment:
Why move this instead of just pointing the config at the `dockstore-integration-testing/src/test/resources/dstesting_pcks8.pem` location?

Reply:
This is because on travis, to my knowledge, we do not know the exact location of the file. We know the relative path, though we don't know what it is relative to. Doing this means that we are using an absolute path that is known.

Reply:
Oh, it should be relative to `TRAVIS_BUILD_DIR`
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables